### PR TITLE
add: tombstoned status

### LINKED ISSUE #600: Base: validator status tombstone
Now that `tombstoned` is available in bdjuno we want to show users if the validator is unbonded or if they are actually tombstoned

### PR DESCRIPTION
**Pull request by ryuash**
## Description

closes #600 

## Checklist
- [x] Ran Linting
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [x] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by ryuash** [comment]
update: check the tombstone and jailed status before 1,2,3. 
You can be tombstoned and unbonding at the same time

File: src/screens/account_details/components/staking/components/delegations/components/desktop/index.tsx [hunk 1]
```diff
@@ -30,7 +30,11 @@ const Desktop: React.FC<{
   const formattedItems = items.map((x) => {
     const amount = formatNumber(x.amount.value, x.amount.exponent);
     const reward = formatNumber(x.reward.value, x.reward.exponent);
-    const statusTheme = getValidatorStatus(x.validatorStatus.status, x.validatorStatus.jailed);
+    const statusTheme = getValidatorStatus(
```
Comment (yayay927) [inline src/screens/account_details/components/staking/components/delegations/components/desktop/index.tsx:33]: Please update test.

File: src/screens/account_details/components/staking/components/delegations/components/desktop/index.tsx [hunk 2]
```diff
@@ -30,7 +30,11 @@ const Desktop: React.FC<{
...
+      x.validatorStatus.status,
+      x.validatorStatus.jailed,
+      x.validatorSigningInfo.tombstoned,
```
Comment (yayay927) [inline src/screens/account_details/components/staking/components/delegations/components/desktop/index.tsx:36]: Please update test.

File: src/screens/account_details/components/staking/components/delegations/components/mobile/index.tsx [hunk 3]
```diff
@@ -24,7 +24,11 @@ const Mobile: React.FC<{
   return (
     <div className={classnames(className)}>
       {items.map((x, i) => {
-        const statusTheme = getValidatorStatus(x.validatorStatus.status, x.validatorStatus.jailed);
+        const statusTheme = getValidatorStatus(
+          x.validatorStatus.status,
+          x.validatorStatus.jailed,
+          x.validatorSigningInfo.tombstoned,
+        );
```
Comment (yayay927) [inline src/screens/account_details/components/staking/components/delegations/components/mobile/index.tsx:31]: Same for this one.

**Review by yayay927 — CHANGES_REQUESTED** [review]

File: src/screens/validators/components/list/components/desktop/index.tsx [hunk 4]
```diff
@@ -42,7 +42,7 @@ const Desktop: React.FC<{
   } = useGrid(columns);
 
   const formattedItems = props.items.map((x, i) => {
-    const status = getValidatorStatus(x.status, x.jailed);
+    const status = getValidatorStatus(x.status, x.jailed, x.tombstoned);
```
Comment (yayay927) [inline src/screens/validators/components/list/components/desktop/index.tsx:45]: Please update test.

File: src/screens/validators/components/list/components/mobile/index.tsx [hunk 5]
```diff
@@ -30,7 +30,7 @@ const Mobile: React.FC<{
   } = useList();
 
   const formattedItems = items.map((x, i) => {
-    const status = getValidatorStatus(x.status, x.jailed);
+    const status = getValidatorStatus(x.status, x.jailed, x.tombstoned);
```
Comment (yayay927) [inline src/screens/validators/components/list/components/mobile/index.tsx:33]: Same for this one.

**Review by yayay927 — CHANGES_REQUESTED** [review]

**Review by yayay927 — APPROVED** [review]

**Pull request merged**